include/rmm/device_buffer.hpp

```diff
@@ -384,12 +393,16 @@ class device_buffer {
    * will be used for deallocation in the `rmm::device_uvector` destructor.
    * However, if either of `resize()` or `shrink_to_fit()` is called after this,
    * the later stream parameter will be stored and used in the destructor.
+   *
+   * @param stream The stream to use for deallocation
    */
   void set_stream(cuda_stream_view stream) noexcept { _stream = stream; }
 
   /**
    * @brief Returns pointer to the memory resource used to allocate and
    * deallocate the device memory
+   *
+   * @return Pointer to the memory resource used to allocate and deallocate the
```

Comment:
Incomplete sentence?
```suggestion
   * @return Pointer to the memory resource used to allocate and deallocate
```

Reply:
Both sentences in this docstring need to end with a full stop.

Reply:
I think the brief does but the return doesn't. Fixed the typo.